jax/api.py

```diff
@@ -108,6 +108,8 @@ def jit(fun: Callable, static_argnums: Union[int, Iterable[int]] = (),
       function with different values for these constants will trigger
       recompilation. If the jitted function is called with fewer positional
       arguments than indicated by ``static_argnums`` then an error is raised.
+      Note that arguments that are not arrays, scalars, or (nested) standard
```

Comment:
This isn't quite accurate, because the set of container types is extensible (although admittedly the existing language doesn't really speak to that point.) However, I think less is more here; you can just say "containers of arrays".

Remove "note that"; it's a phrase that never really adds anything. Perhaps:

```
Arguments that are not arrays or containers thereof must be marked as static.
```

Reply:
Done.